### ISSUE TITLE
OpenType variable font?

### ISSUE THREAD

**Issue by dpk** [Fuzzypeg/Coelacanth#18]
You have a very impressive selection of master files and interpolation tools and, on the whole, a beautiful project! I’m very much looking forward to seeing your future progress and further releases.

Would it be possible to use your existing masters and interpolation data to build single font files for the Roman and Italic styles respectively, using OpenType 1.8 font variations for both optical sizing and weight? This could significantly reduce the size of the font files and also free users from the fixed set of optical sizes.

**Comment by Fuzzypeg** [comment]
Yes, I'm working toward this. This repo is moving to https://gitlab.com/Fuzzypeg/coelacanth where I'm working on more automation with a view to soon generating a variable font. I'll recreate the ticket there.
